src/components/scenes/FioAddressSettingsScene.js

```diff
@@ -115,14 +156,33 @@ class FioAddressSettingsComponent extends React.Component<Props, LocalState> {
 
   render() {
     const { route } = this.props
-    const { fioAddressName, fioWallet } = route.params
-    const { showTransfer } = this.state
+    const { fioAddressName, fioWallet, bundles } = route.params
+    const { showTransfer, showAddBundles } = this.state
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name